src/components/Button/Button.tsx

```diff
@@ -96,6 +102,14 @@ class Button extends UIComponent<IButtonProps, any> {
     as: 'button',
   }
 
+  constructor(props, state) {
+    super(props, state)
+    const accBehavior: string = props.accBehavior
+    this.accBehavior = AccBehaviorFactory.getBehavior(
+      AccBehaviorType[accBehavior] || AccBehaviorType.button,
+    )
```

Comment:
Seems we should set `defaultProps.accBehavior` to `button` and similar for all components.

Reply:
Per convo, accessibility can actually change depending on props (e.g. vertical menu vs a normal menu).  Because props can change overtime, we need to compute accessibility at render rather than at construction.

Let's move this into `renderComponent` just as we're calculating `classes` on each render.